TAS_T3DAdapter/TAST3DAdapter.cs

```diff
@@ -134,7 +134,7 @@ public override IEnumerable<object> Pull(IRequest request, Dictionary<string, ob
             }
             catch (Exception e)
             {
-                ErrorLog.Add(e.ToString());
+               Engine.Reflection.Compute.RecordError(e.ToString());
```

Comment:
For consistency with other BEnv toolkits and TAS Toolkit itself, could we append `BH.` to the front of all the `RecordError` messages.